fusor-ember-cli/app/routes/deployment.js

```diff
@@ -133,20 +133,15 @@ export default Ember.Route.extend(DeploymentRouteMixin, UsesOseDefaults, {
           "Authorization": "Basic " + self.get('session.basicAuthToken')
         }
       }).then(
-        function (response) {
-          var uuid = response.id;
-          console.log('task uuid is ' + uuid);
-          deployment.set('foreman_task_uuid', uuid);
-          deployment.save().then(
-            function () {
-              controller.set('showSpinner', false);
-              return self.transitionTo('review.progress.overview');
-            },
-            function () {
-              controller.set('showSpinner', false);
-              controller.set('errorMsg', 'Error in saving UUID of deployment task.');
-              controller.set('showErrorMessage', true);
-            });
+        function (_) {
```

Comment:
Do you mean just setting foreman_task_uuid to the value returned in the response, or somehow refreshing the deployment record in ember data with the response JSON obj? I recall seeing something about the latter, but I don't remember what it was. The former will dirty the object.